README.md

```diff
@@ -238,10 +238,10 @@ bool onWatchList = amlResult.IsOnWatchList();
 1) Navigate to the [src/Examples/CoreExample](src/Examples/CoreExample) folder
 1) Rename the [.env.example](src/Examples/CoreExample/.env.example) file to `.env`
 1) Fill in the environment variables in this file with the ones specific to your application (mentioned in the [Client initialisation](#client-initialisation) section)
-1) From the Yoti Dashboard, set the Callback URL of your application to be `localhost:55555/account/connect`. You'll need to change it from `https` to `http` once the webpage loads, as the Dashboard doesn't allow `http` URLs
+1) From the Yoti Dashboard, set the Callback URL of your application to be `localhost:44321/account/connect`.
```

Comment:
... set the application domain to `localhost:44321` and the scenario callback URL to `account/connect`